src/Layer/Layer.svelte

```diff
@@ -0,0 +1,45 @@
+<script>
+  import { getContext, setContext } from "svelte";
+
+  /**
+   * Specify the layer level and override any existing levels based on hierarchy.
+   * @type {0 | 1 | 2 }
+   */
+  export let level = undefined;
+
+  /**
+   * Specify the HTML element to render. If none is specified, a `div` is rendered.
+   * @type {typeof import("svelte").SvelteComponent}
+   */
+  export let as = "div";
+
+  /**
+   * Specify the Layer HTML element props
+   * @type {import('svelte/elements').HTMLElementAttributes}
+   */
+  export let layerProps = {};
```

Comment:
Since there is only one element I would stick to spreading `$$restProps`.

Is there a consensus how to handle this?

Reply:
@metonym @theetrain do you have a view on this?

Reply:
For v1, we want to avoid `$$restProps`, per #1621